site/content/blogs/release-v130.en.md

```diff
@@ -81,6 +90,10 @@ stages:
 
 ## `copilot deploy` enhancements
 
+## `detach flag`
```

Comment:
Similar to the other title, I think we should make this one more descriptive. For example. `Use `--detach` to deploy without waiting`.